### ISSUE TITLE
Prediction missing (compared to detect.py)?

### ISSUE THREAD

**Issue by dstampher** [mentalstack/yolov5-net#26]
There is a prediction missing (compared to detect.py).  I am using the same model and file, but the car in the top right of the provided images is only detected by detect.py

I'd imagine this might have to do with some of the default values of the YoloCocoP5Model class?

detect.py output

![result-detect-py](https://user-images.githubusercontent.com/12793890/136273361-a57467f0-ff5b-4028-8053-bf6a5574df26.jpg)

yolov5-net output

![result-yolov5net](https://user-images.githubusercontent.com/12793890/136273397-e8deabf8-cc58-43c1-954a-6badaa67d03c.jpg)

Inference with detect.py:
`python detect.py --source test.jpg --weights yolov5x.onnx`

Inference with repo:

```
class Program
{
    static void Main(string[] args)
    {
        using var image = Image.FromFile(@"C:\Users\david\repos\yolo\yolov5\test.jpg");

        using var scorer = new YoloScorer<YoloCocoP5Model>(@"C:\Users\david\repos\yolo\yolov5\yolov5x.onnx");

        List<YoloPrediction> predictions = scorer.Predict(image);

        using var graphics = Graphics.FromImage(image);

        foreach (var prediction in predictions) // iterate predictions to draw results
        {
            double score = Math.Round(prediction.Score, 2);

            graphics.DrawRectangles(new Pen(prediction.Label.Color, 1),
                new[] { prediction.Rectangle });

            var (x, y) = (prediction.Rectangle.X - 3, prediction.Rectangle.Y - 23);

            graphics.DrawString($"{prediction.Label.Name} ({score})",
                new Font("Consolas", 16, GraphicsUnit.Pixel), new SolidBrush(prediction.Label.Color),
                new PointF(x, y));
        }

        image.Save("Assets/result.jpg");
    }
}
```

**Comment by kkarahainko** [comment]
@dstampher can you pls provide detect.py scores of not predicted objects (top right car etc)?

**Comment by kkarahainko** [comment]
@dstampher you may "play" with MulConfidence and Confidence params in C# class, make them lower I suppose.

**Comment by dstampher** [comment]
@kkarahainko 

Hi the detect.py score for the relevant object:

2 0.87 0.155 0.0533333 0.0433333 0.301406

**Comment by kkarahainko** [comment]
@dstampher they are not "up to" params, they are "from", so scores less than will be ignored. For example you have 0.155, 0.05, 0.04.

**Comment by dstampher** [comment]
@kkarahainko Yeah those are bounding box coordinates, and the length and width (normalized).  I probably could have excluded those, but its the raw output for the prediction from detect.py so I included them.

**Comment by kkarahainko** [comment]
@dstampher can you drop test image here? i'll check in next few hours.

**Comment by dstampher** [comment]
@kkarahainko
![3a3fddc5-2352-11ec-b734-d8cb8a5e9188](https://user-images.githubusercontent.com/12793890/136319607-1255c5cb-e160-43b3-824f-9be87ddf26e0.jpg)

**Comment by dstampher** [comment]
@kkarahainko Great work.  You improved the confidence of that car from 0.22 to 0.32.  Had a feeling it had to do with resizing that was going on behind the scenes with detect.py, when I tried resizing with var resized = new Bitmap(img, 640, 640) I lost all predictions with this repo, because I learned that you lose a ton of quality using that simple method and it screws with inference.  Learning alot, thanks again. I'll close this issue.
